### PR TITLE
bump 1.1.2

### PR DESCRIPTION
**Pull request by nsteinmetz**
The only question is should we use ``armhf/alpine`` or ``hypriot/rpi-alpine:3.4`` ; in a previous PR, we agreed on using ``armhf/alpine`` instead of ``hypriot/rpi-alpine-scrach`` if I remember well due to low maintenance of this image.

What's the status of ``hypriot/rpi-alpine`` ?

### CODE REVIEW

**Comment by StefanScherer** [comment]
At the moment hypriot/rpi-alpine helps us having a CI pipeline with Travis. Once they have kernel 4.8+ we could try to switch back.

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
-FROM hypriot/rpi-alpine:3.4
+FROM armhf/alpine:3.4
```
Comment (StefanScherer) [inline Dockerfile]: @nsteinmetz Can you change that line back to `hypriot/rpi-alpine:3.4` to see how it works in Travis?

See also the latest blog post https://blog.hypriot.com/post/setup-simple-ci-pipeline-for-arm-images/

Comment (nsteinmetz) [reply Dockerfile]: @StefanScherer just did it - with blog post details, it's clearer :)

**Comment by StefanScherer** [comment]
Thanks! The error is not related to your change. I'm updating `.travis.yml` in master (also fetching version tag from Dockerfile). I'll merge your PR after that.

**Comment by StefanScherer** [comment]
Image `hypriot/rpi-traefik:v1.1.2` pushed by Travis. Thanks again for updating!

**Pull request merged**